pkgs/development/libraries/sqlite/analyzer.nix

```diff
@@ -6,11 +6,11 @@ in
 
 stdenv.mkDerivation rec {
   pname = "sqlite-analyzer";
-  version = "3.30.0";
+  version = "3.30.1";
 
   src = assert version == sqlite.version; fetchurl {
```

Comment:
Would it make sense to use `inherit (sqlite) version src;` instead? And adding a comment to the top of sqlite file to make sure the analyzer builds and to `passthru.tests`?

Reply:
sqlite `src` doesn't contain the tcl tool. Maybe we should make sqlite `src` the full source zip? Then we can make this change.

Reply:
@jpathy Do you want to do it as a followup PR, fixing this assertion failure first?

Reply:
Sure. let's get this merged.